src/Api/CanadaPost.php

```diff
@@ -40,44 +51,63 @@ public function __construct(
   }
 
   /**
+   * Get username.
+   *
    * @return string
+   *   The username.
    */
   public function getUsername() {
     return $this->username;
   }
 
   /**
+   * Set the username.
+   *
    * @param string $username
+   *   The username.
    */
   public function setUsername($username) {
     $this->username = $username;
   }
 
   /**
+   * Get password.
+   *
    * @return string
+   *   The password.
    */
   public function getPassword() {
     return $this->password;
   }
 
   /**
+   * Set the password.
+   *
    * @param string $password
+   *   The password.
    */
   public function setPassword($password) {
     $this->password = $password;
   }
 
   /**
+   * Get customer number.
+   *
    * @return string
+   *   The customer number.
    */
   public function getCustomerNumber() {
     return $this->customerNumber;
   }
 
   /**
+   * Set the customer number.
+   *
    * @param string $customerNumber
+   *   The customer number.
    */
   public function setCustomerNumber($customerNumber) {
     $this->customerNumber = $customerNumber;
   }
+
 }
```

Comment:
The client for making requests to the Canada Post API is now provided by the SDK. This class is not needed.